core/tests/ts-integration/tests/api/web3.test.ts

```diff
@@ -808,6 +808,76 @@ describe('web3 API compatibility tests', () => {
         ).toBeRejected();
     });
 
+    test('Should check transaction signature', async () => {
+        const CHAIN_ID = +process.env.CHAIN_ETH_ZKSYNC_NETWORK_ID!;
```

Comment:
nit: you can get chain id from API

Reply:
imo, its better to use env var for tests